docs/docs/Enterprise/audit_logs.md

```diff
@@ -39,17 +39,17 @@ The dropdown will list all the apps present in your account. Choose an app to fi
 | USER_INVITE | You can invite users to your account from `Manage Users` section and an event is audited everytime an invite is sent. |
 | USER_INVITE_REDEEM | This event is recorded whenever an invite is redeemed. |
 | APP_CREATE | This event is recorded when a user creates a new app. |
-| APP_UPDATE | This event is recorded whenever actions like renaming the app, making the app public, editing shareable link, or deploying the app are made. |
-| APP_VIEW | This event is logged when someone views the launched app. (public apps isn't accounted for) |
+| APP_UPDATE | This event is recorded whenever actions like renaming the app, making the app public, editing shareable links, or deploying the app are made. |
```

Comment:
Please revert this.

Reply:
Revert change on line no 42 and 43 right?
Thank you

Reply:
Just the 42 one.

Reply:
Done!

Reply:
@withshubh Thank you! for approving these changes.
I have a questions.
This PR is not merged, will this still be counted in Hacktoberfest?